### DISCUSSION TITLE
Use the pydantic alias feature

### DISCUSSION THREAD

**Discussion by wu-clan** [fastapi-practices/fastapi_best_architecture#91] [Ideas]
In python, underscore naming is the recommended scheme in PEP8, but in other languages it is common to use hump naming, which is indirectly enabled using the pydantic alias setting.

1. alias_generator
```python
class Config:
    @classmethod
    def alias_generator(cls, string: str) -> str:
        # this is the same as `alias_generator = to_camel` above
        return ''.join(word.capitalize() for word in string.split('_'))
```

2. Field
```
is_example = Field(alias='IsExample')
```

**Comment by downdawn** [comment]
This should be on demand

**Reply by wu-clan** [reply]
Yes, it is worth noting that the use of alias will affect Swagger，pydantic.dict()…
